flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+    "only-arches": [
+        "x86_64"
```

Comment:
It's MIT licensed, so why restricting to x86_64?

Reply:
Arm is not yet supported by upstream attempting to build it for arm results in build errors. There is work being done to fix this

Reply:
I thought that Electron was all about being cross-platform.... *sigh*

Reply:
> I thought that Electron was all about being cross-platform.... _sigh_

👋🏻   Upstream maintainer here: we're not supporting other architectures currently due to the native modules and Git needing to target these platforms, not Electron itself - https://github.com/shiftkey/desktop/issues/251 has more context on what's needed for ARM64 support, for example...